repos/repo_url.go

```diff
@@ -56,12 +56,28 @@ func (r *RepoURL) String() string {
 
 // Set parses a URL string into RepoURL struct.
 func (r *RepoURL) Set(s string) error {
+	var t string
+
+	const two = 2
+	const three = 3
+
+	c := strings.Split(s, "/")
+
+	switch l := len(c); {
+	// this will takes care for repo/owner format, by default use github.com
```

Comment:
nit: start senetence with capitale letter and end with fulldot. Some checks may fail.
Also fix https://github.com/ossf/scorecard/pull/872/checks?check_run_id=3371804644

Reply:
Done.